src/Pages/Auth/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import { background4 } from "@/Assets/Images";
+import { FaCheckCircle, FaEnvelope, FaKey} from "react-icons/fa";
+import { FaRegCircleXmark } from "react-icons/fa6";
+
+export default function ChangePassword() {
+  return <>
+   <div className="bg-mainBg h-screen">
+        <div className="container mx-auto p-5">
+           {/* Top-left Logo */}
+           <div className="text-white my-3 w-40 flex flex-row p-2">
+            <div className="flex">
+              <FaRegCircleXmark className="text-3xl" />
+              <FaCheckCircle className="text-3xl" />
+            </div>
+            <div className="text-lg">| Quizwiz</div>
```

Comment:
The whole logo should be  wrapped inside an anchor element that redirects to the base URL.